pkg/nsx/services/mediator/mediator.go

```diff
@@ -67,8 +67,21 @@ func (serviceMediator *ServiceMediator) GetAvailableSubnet(subnetSet *v1alpha1.S
 			return *nsxSubnet.Path, nil
 		}
 	}
+	var tags []model.Tag
+	findLabelDefaultPodSubnetSet := false
+	for k, v := range subnetSet.Labels {
+		if k == common.LabelDefaultSubnetSet && v == common.LabelDefaultPodSubnetSet {
```

Comment:
we only check there is LabelDefaultPodSubnetSet on subnet, but  LabelDefaultVMSubnet  and LabelDefaultPodSubnetSet lables only exit for defaultSubnetset, we also need to handle user created subnet and subnet under user defined subnetset.

Reply:
@dantingl said apart from LabelDefaultPodSubnetSet, others are all VM, right？

Reply:
Yes, only default SubnetSet CR for Pod